### ISSUE TITLE
Blacklist/Whitelist Identities through config

### ISSUE THREAD

**Issue by PalgamingHD** [Draylar/identity#190]
Looking through configs theres no way to disable certain identities, would be really useful in modpacks.

**Comment by Sunconure11** [comment]
I agree with this, as old as this issue is. Some modded mobs bug out with identity. @Draylar

**Comment by ScoutTheSaneless** [comment]
i strongly agree with this, as i just encountered one said issue
